src/commands/start-client.js

```diff
@@ -137,7 +137,15 @@ module.exports = function (buildOnly) {
   }
   else {
     logger.info("Bundling assets…");
-    compiler.run(() => {
+    compiler.run((error, stats) => {
```

Comment:
most likely yes, I just wanted to rush a fix in since production builds are currently broken